test/e2e/multicast_test.go

```diff
@@ -46,6 +46,7 @@ func skipIfMulticastDisabled(tb testing.TB, data *TestData) {
 }
 
 var igmpQueryType = int32(0x11)
+var transportInterface string
```

Comment:
don't see why it must use a global variable, having such variable introduces potential data race, makes tests not self-contained. It's also inconsistant why only this variable is made so. Just pass it to required callees like all other variables.

Reply:
updated